roles/lib/files/FWO.Services/NetworkObjectComparer.cs

```diff
@@ -18,12 +18,18 @@ public bool Equals(NetworkObject? nwObject1, NetworkObject? nwObject2)
                 return false;
             }
 
-            return (!option.NwRegardIp || nwObject1.IP == nwObject2.IP && nwObject1.IpEnd == nwObject2.IpEnd)
-                && (!option.NwRegardName || nwObject1.Name == nwObject2.Name);
+            return (!option.NwRegardIp || (string.Equals(nwObject1.IP, nwObject2.IP, StringComparison.Ordinal)
```

Comment:
here using case sensitive, in appServerComparer case insensitive. Maybe we should think about centralizing /harmonizing this setting